web/src/styles/app/alert.css

```diff
@@ -1,25 +1,36 @@
 .alert {
   background-color: #eee;
-  border-left: 0.5rem solid #ccc;
   border-radius: 3px;
+  padding: 0.9rem 1.95rem 1.5rem;
+}
+
+.alert::before {
+  width: 0;
+  height: 0;
+  content: "";
+  display: block;
+  border-left: 12px solid transparent;
+  border-right: 12px solid transparent;
+  transform: translate(-2.4rem, -.95rem) rotate(-45deg);
```

Comment:
I'd love to understand where these translation numbers come from  38.4px to the left and 15.2px up?  (If I understand correctly that 1rem is 16px, anyway.)  Is this to back out the left/top paddings plus some trigonometry due to the rotation?

Reply:
Yep! Although `trigonometry` in this case is actually `adjusting position a hundreth at a time with high contrast colors in dev tools`.